### ISSUE TITLE
Autonomy

### ISSUE THREAD

**Issue by amaurymartiny** [chronoscio/backend#41]
I want to move the conversation here too, so that it's logged and people can start working on it.

The simplest idea we can implement today is Bapo's slider idea, and I propose we implement this in v1. In parallel, we can think of a more robust and general weighing system for later versions.

More concretely, here are the steps:
1. Group all the political entity types in [this gdoc](https://docs.google.com/spreadsheets/d/1lYX3tjX7DY0fsSadBmtzd6OSG3DMXi-irEQA5H2J9eA/edit#gid=2108598426) into 5 or 6 big categories, that will be shown on the same autonomy level. Mappers can help here.
2. Add a query parameter in the url `?autonomy=2` to retrieve only territories that are in that autonomy level.

![image](https://user-images.githubusercontent.com/1293565/46246561-d7395f80-c3f7-11e8-8ed3-55419915f816.png)

**Comment by amaurymartiny** [comment]
Right, I was thinking about b in the beginning, but I agree with you, a is a better option.

[DiplomaticRelations (annex 2)](https://docs.google.com/spreadsheets/d/1lYX3tjX7DY0fsSadBmtzd6OSG3DMXi-irEQA5H2J9eA/edit#gid=506309222) each have parents and children. So I guess the best way to render this to to decide on which level each parent/child of each DiplomaticRelation will be rendered.

Something like -

| Level | Renders |
| ----- | ------- |
| 1 | `Military alliance Parent`, `Codominium Parent` |
| 2 | `Military alliance Child` etc |

@bapo224 Would this model work?

**Comment by bapo224** [comment]
Yes I think that will work, but it might be a bit more complicated for example a subject nation that has a subject of his own. That nation would be both client state parent and client state child.

Maybe a third possibility is making the render level a parameter the mappers have to fill in rather than something the site has to figure out?

**Comment by amaurymartiny** [comment]
I would not recommend a render level parameter inputted by the mappers, because it creates duplicate and inconsistent data. The only thing a mapper should care about is objective data (how is a PoliticalEntity related to another?) rather than functionality (how should this data be rendered?).

I the above table, instead of only having Parent/Child, we could replace by Gen0, Gen1 to represent generations. I'm pretty sure we can figure out something.

**Comment by whirish** [comment]
@amaurymartiny @bapo224 any updates on this?
